src/main/Duke/virtue/Command.java

```diff
@@ -0,0 +1,135 @@
+package virtue;
+
+public class Command {
+    // The available types of commands in Virtue.
+    public enum CommandType {
```

Comment:
Using an enum to define CommandType was a good idea, since it groups together related constants, and makes the code more readable